### ISSUE TITLE
Error : Call to undefined function GeneaLabs\LaravelModelCaching\Traits\emptyArray()

### ISSUE THREAD

**Issue by latheesan-k** [GeneaLabs/laravel-model-caching#79]
# Issue
I am running a test using eloquent model and during test, I am getting this error:

> Error : Call to undefined function GeneaLabs\LaravelModelCaching\Traits\emptyArray()

## Environment
Laravel Version: *v5.5.34*
Laravel Model Caching Package Version: *0.2.26*
PHP Version: *PHP 7.1.11 (cli) (built: Oct 25 2017 21:07:06) ( ZTS MSVC14 (Visual C++ 2015) x86 )*
Homestead Version: *N/A I am using XAMPP*
Operating System & Version: *Windows 10 x64*

## Stack Trace
```
Testing started at 23:37 ...
C:\xampp\php\php.exe C:/xampp/htdocs/myApp/vendor/phpunit/phpunit/phpunit --no-configuration Tests\Feature\Api\V1\OrganisationApiTest C:\xampp\htdocs\myApp\tests\Feature\Api\V1\OrganisationApiTest.php --teamcity
PHPUnit 6.5.6 by Sebastian Bergmann and contributors.


Error : Call to undefined function GeneaLabs\LaravelModelCaching\Traits\emptyArray()
 C:\xampp\htdocs\myApp\vendor\genealabs\laravel-model-caching\src\Traits\Cachable.php:44
 C:\xampp\htdocs\myApp\vendor\genealabs\laravel-model-caching\src\Traits\Cachable.php:75
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php:350
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php:200
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php:173
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Concerns\HasEvents.php:148
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php:598
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php:569
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Database\Eloquent\FactoryBuilder.php:172
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Support\Collection.php:339
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Database\Eloquent\FactoryBuilder.php:173
 C:\xampp\htdocs\myApp\vendor\laravel\framework\src\Illuminate\Database\Eloquent\FactoryBuilder.php:153
 C:\xampp\htdocs\myApp\tests\TestCase.php:28
 C:\xampp\htdocs\myApp\tests\TestCase.php:18
 C:\xampp\htdocs\myApp\tests\Feature\Api\V1\BaseApiTestCase.php:21
 C:\xampp\htdocs\myApp\tests\Feature\Api\V1\OrganisationApiTest.php:11
 


Time: 1.54 seconds, Memory: 14.00MB


ERRORS!
Tests: 1, Assertions: 0, Errors: 1.

Process finished with exit code 2
```

P.S. This works fine on version `0.2.25`

**Comment by mikebronner** [comment]
Hmm, this might be a Laravel 5.6 thing, I will have to investigate. Sorry for the troubles. Will have a solution shortly!

**Comment by mikebronner** [comment]
This is fixed in version 0.2.27. Thanks again for reporting this! :)

**Comment by latheesan-k** [comment]
Amazing turn-around, +1 Thanks.
